client/ayon_core/pipeline/workfile/workfile_template_builder.py

```diff
@@ -735,6 +740,16 @@ def populate_scene_placeholders(
 
                 placeholder.set_finished()
 
+            # Trigger on_depth_processed event
+            self.trigger_event(
```

Comment:
```suggestion
            self.emit_event(
```

Reply:
Thanks - seems I missed this in the refactor. Fixed now.